docs/configuration-ini.md

```diff
@@ -255,6 +255,48 @@ bias_initializer = zeros
 Activation = softmax
 ```
 
+### Backbone section
+
+Describe another model, termed as backbone, to be used used in the model described by the current ini file.
+The backbone to be used can described with another ini configuration file path, or with model file for external frameworks.
+Support for backbones of external framework for Tensorflow-Lite is provided natively with Tensorflow-Lite framework.
+Support for backbones of other external frameworks is done using nnstreamer and its plugin.
+When using nnstreamer for external framework, ensure to add the corresponding baseline ML framework and its corresponding nnstreamer plugin as a dependency or install manually.
+For example, when using PyTorch based model as a backbone, both the packages *PyTorch* and *nnstreamer-pytorch*.
+
+Backbones made of nntrainer models, described using ini, support training the backbone also.
+However, this is not supported with external frameworks.
+It is possible to describe a backbone inside a backbone ini configuration file, as well as listing down multiple backbones to build a single model.
+For backbone ini configuration file, Model and Dataset sections are ignored.
+
+Describing a backbone is very similar to describing a layer.
+Start with a "[ ${layer name} ]" which must be unique throughtout the model. In case of backbone, the name of the backbone is preprended to the name of all the layers inside the backbone.
+
+1. ```backbone = <string>```
+
+   Path of the backbone file. Supported model files:
+    * .ini - NNTrainer models
+    * .tflite - Tensorflow-Lite models
+    * .pb / .pt / .py / .circle etc via NNStreamer (corresponding nnstreamer plugin required)
+
+2. ```trainable = <bool>```
+
+   If this backbone must be trained. Only supported for ini backbones (nntrainer models).
+``
+Below is sample backbone section.
+
+```ini
+# block1
```

Comment:
Could you add the omitted Model section and other simple layers? I think it would be hard to understand where this goes.

Reply:
done.